opengrok-web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -233,6 +239,14 @@
         <url-pattern>/eforbidden</url-pattern>
         <!-- FORBIDDEN -->
     </servlet-mapping>
+    <servlet>
+        <servlet-name>metrics</servlet-name>
+        <servlet-class>com.codahale.metrics.servlets.AdminServlet</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>metrics</servlet-name>
+        <url-pattern>/metrics/*</url-pattern>
```

Comment:
can this be part of the rest api?
also authorization applies or not?

Reply:
what about the Statistics endpoint documentation in Apiary ? This will probably need to be updated.

Reply:
As for the authentication/authorization this is definitely a concern since the metrics included names of projects along with their success/failure authorization counts. Surely we do not want to leak project names without authorization.

Reply:
According to https://metrics.dropwizard.io/3.1.0/manual/servlets/ `AdminServlet` basically aggregates bunch of servlets, including the `MetricServlet` that exposes the `/metrics` endpoint which presents the metricts. Maybe put this behind `LocalhostFilter` ?

Reply:
Actually, you can leave out the project names from the meters altogether since it is not good practice anyway - it would cause "cardinality explosion" (https://youtu.be/deBSrmskpcY?t=1728)

Reply:
Once the project names are out, the endpoint does not have to be covered by authentication/authorization I think. Otherwise #3222 would help.